src/backend/gporca/data/dxl/minidump/CTAS_OrderedAgg_multiple_cols.mdp

```diff
@@ -1808,7 +1808,7 @@ EXPLAIN CREATE TABLE test AS
         </dxl:LogicalProject>
       </dxl:LogicalCTAS>
     </dxl:Query>
-    <dxl:Plan Id="0" SpaceSize="16799099328000">
```

Comment:
What is SpaceSize? Why the given patch affects this query plan?

Reply:
it is count of alternatives https://github.com/arenadata/gpdb/blob/46f40f274b7ebcc5a8d42076d49c786849931d47/src/backend/gporca/libgpopt/src/engine/CEngine.cpp#L694-L695

Reply:
Why this patch decrease alternatives count? These alternatives were incorrect? Can you proof it?

Reply:
> Why this patch decrease alternatives count?

not only descrease: https://github.com/arenadata/gpdb/pull/434/files#diff-9040fc5f4b8fccdcc163c373e9325c1781d11028d016f6ff88213c6b09e5a366L511

Reply:
I return extra condition and only 2 mpd are affected now

Reply:
That doesn't answer's Vasiliy's question at all.

Reply:
> That doesn't answer Vasiliy's question at all.

given patch does not affect this query plan any more

Reply:
is this question resolved?

Reply:
For me, I didn't check latest commits to see why mdp files are unchanged again.